FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ButtonPrototypeVC.swift
+//  FightPandemics
+//
+//  Created by Luciano Schillagi on 5/16/20.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class ButtonPrototypeController: UIViewController {
+
+    private var regularButton = PillButton()
+    private var boldButton = PillButton()
+    private var outlineButton = PillButton()
+    private var inactiveButton = PillButton()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .yellow
+        setupRegularButton()
+        setupBoldButton()
+        setupOutlineButton()
+        setupInactiveButton()
+  }
+
+    func setupRegularButton() {
+         view.addSubview(regularButton)
+         regularButton.setTitle("Regular Button", for: .normal)
+         regularButton.translatesAutoresizingMaskIntoConstraints = false
+         regularButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
+         regularButton.widthAnchor.constraint(equalToConstant: 280).isActive = true
+         regularButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+         regularButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -500).isActive = true
```

Comment:
for constraints, we're trying to be consistent using one utility. example usage: https://github.com/FightPandemics/FightPandemics-iOS/pull/87/files#diff-5fbf4ba7b8989796d3ea7e5fd6f77722R51

Reply:
Ok. Question: Which framework does it belongs "makeSubview(of:)" to?